### DISCUSSION TITLE
i tried to rip tendering of repo, it appears impossible

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [informalsystems/ibc-rs#805] [General]
because its thing leaked into other more generically named directories

tbh may be make new crate ibc-rs-core, and start moving tendermint free types and code here, with reexport for a while.

tenderming is heavy dependency (and seems brittle)

also it will provide very good example and proof that complicated clients can live out of same crate.

**Comment by Farhad-Shabani** [comment]
Totally understand. This is on our radar: https://github.com/cosmos/ibc-rs/issues/716. 
There are some prerequisites for being there. We will try to place a greater priority on this.

**Reply by dzmitry-lahoda** [reply]
oh, did not found it. thank you. saw only case for leaving tendermint in crate some time ago.
